src/commands/NewIntegrationFileCommand.ts

```diff
@@ -47,7 +47,9 @@ export async function create() : Promise<void> {
 			if (filename) {
 				const kamelExe = kamel.create();
 				const newFileFullPath: string = computeFullpath(language, workspaceFolder, filename);
-				kamelExe.invoke(`init "${newFileFullPath}"`);
+				await kamelExe.invoke(`init "${newFileFullPath}"`);
```

Comment:
Maybe its worth to try one of the available commands to open a file. 

workbench.action.files.openFile
workbench.action.focusQuickOpen
workbench.action.openEditorAtIndex

Reply:
using a command won't break compilation if it is removed/modified/parameters changed.

Reply:
vscode.openFolder works great -- just pass it the path, which we already have

Reply:
`workbench.action.files.openFile` is opening local file browser to pick a file. (tried with a string and an uri as parameter)
`workbench.action.focusQuickOpen` is opening a local file browser to pick a file when providing an URI and do nothing with a string

also, when using commands, how to know which parameters need sot be provided? Looking into the source code is really nto convenient.

Reply:
As discussed on our team call, maybe we go with the API approach unless there's good reason to use a command in this case.